AdamSensors/V1/_labels.yaml

```diff
@@ -0,0 +1,2 @@
+# Labels for recommended settings; a dict of label: config_file
+test: test.yaml
```

Comment:
Add new line at the end of the file....

Reply:
I know we have some around but we're trying to avoid labels like "test". 
Can you try to use a more descriptive label? If this is for a particular setup, like in your office, you could use something like `tucson_test_2020` or something on those line...